apps/api/src/app/widgets/e2e/initialize-widget-session.e2e.ts

```diff
@@ -99,3 +87,22 @@ async function initWidgetSession(subscriberId: string, session, hmacHash?: strin
     hmacHash: hmacHash,
   });
 }
+
+async function setHmacConfig(session: UserSession) {
+  const integrationRepository = new IntegrationRepository();
+
+  await integrationRepository.update(
+    {
+      _environmentId: session.environment._id,
+      _organizationId: session.environment._organizationId,
```

Comment:
```suggestion
      _organizationId: session.organization._id,
```
This is very stupid suggestion. Feel free to ignore it. I find easier to mentally map it to pick the id from the entity if available, like in the case of the user session.